R/serialize.R

```diff
@@ -66,6 +68,28 @@ serialize_parse_data_flat <- function(pd_flat) {
         collapse = "")) %>%
     .[["text_ws"]] %>%
     strsplit("\n", fixed = TRUE) %>%
-    .[[1L]]
+    .[[1L]] %>%
+    trimws(which = "right")
```

Comment:
Ok, you are right. In fact, it already works like that, just not for one edge case. Empty comments, since we have the rule that adds a space at the beginning of each comment. 
```r
#' 
```
So now I changed the rule to only add a space if the comment is *non-empty* (see adapted help file).